cmd/gateway/s3/gateway-s3.go

```diff
@@ -511,6 +511,12 @@ func (l *s3Objects) CopyObject(ctx context.Context, srcBucket string, srcObject
 		srcInfo.UserDefined[k] = v[0]
 	}
 
+	if _, ok := srcInfo.UserDefined[xhttp.AmzObjectTagging]; ok {
```

Comment:
this change is not needed

Reply:
Yes, will remove this change and go with the change suggested before.